### PR TITLE
Add the baseline feature

### PR DESCRIPTION
**Pull request by larochef**
This allows skipping some init migrations for existing environments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,6 +193,7 @@ Elasticsearch-Evolution can be configured to your needs:
 -   **historyIndex** (default=es_evolution): Name of the history index that will be used by Elasticsearch-Evolution. In this index Elasticsearch-Evolution will persist his internal state and tracks which migration script has already been executed.
 -   **historyMaxQuerySize** (default=1000): The maximum query size while validating already executed scripts. This query size have to be higher than the total count of your migration scripts.
 -   **validateOnMigrate** (default=true): Whether to fail when a previously applied migration script has been modified after it was applied.
+-   **baselineVersion** (default=1.0): Version to use as a baseline. versions lower than it will not be applied.
```
Comment (xtermi2) [inline README.md:196]: you should mention this new feature below in the _changelog_ section (`v0.4.1-SNAPSHOT`), e.g 
```markdown
- It is now possible to set a `baselineVersion` to skip migrations with versions lower than the defined `baselineVersion` ([#164](https://github.com/senacor/elasticsearch-evolution/issues/164))
```

File: elasticsearch-evolution-core/src/main/java/com/senacor/elasticsearch/evolution/core/api/config/ElasticsearchEvolutionConfig.java [hunk 2]
```diff
@@ -129,6 +135,7 @@ public ElasticsearchEvolutionConfig validate() throws IllegalStateException, Nul
             }
             requireNotBlank(historyIndex, "historyIndex must not be empty");
             requireCondition(historyMaxQuerySize, size -> size > 0, "historyMaxQuerySize value '%s' must be greater than 0", historyMaxQuerySize);
+            requireNotBlank(baselineVersion, "baselineVersion must not be empty");
```
Comment (xtermi2) [inline elasticsearch-evolution-core/src/main/java/com/senacor/elasticsearch/evolution/core/api/config/ElasticsearchEvolutionConfig.java:138]: you should do a better validation here, e.g
```java
try {
  MigrationVersion.fromVersion(baselineVersion);
} catch (RuntimeException e) {
  throw new IllegalArgumentException("baselineVersion is invalid", e);
}
```

**Review by xtermi2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (larochef) [reply README.md:196]: Done

File: elasticsearch-evolution-core/src/main/java/com/senacor/elasticsearch/evolution/core/api/config/ElasticsearchEvolutionConfig.java (hunk 2 above)
Comment (larochef) [reply elasticsearch-evolution-core/src/main/java/com/senacor/elasticsearch/evolution/core/api/config/ElasticsearchEvolutionConfig.java]: Done

**Review by xtermi2 — APPROVED** [review]

**Comment by iouakrim** [comment]
Hello ,
thanks @larochef pour this feature ! 
@maaaNu when do you plane to do new release with this feature ? thanks !

**Comment by maaaNu** [comment]
@xtermi2 Can you help us out?

**Pull request merged**